samples/dashboards/demo/project-management/demo.css

```diff
@@ -10,6 +9,14 @@
     --highcharts-color-3: #fc6556;
     --highcharts-color-5: #404040;
 
+    --highcharts-neutral-color-100: #000000;
```

Comment:
Knitpicking: the CSS linter slaps me on the finger for writing #000000 instead of #000 (not important).